tox.ini

```diff
@@ -174,17 +258,13 @@ max-complexity = 60
 
 description = run Mypy (static type checker)
 
-basepython = python3.8
-
-skip_install = True
```

Comment:
Don't skip installing the project (and dependencies) because that's how you get type stubs from dependencies.